config.yaml

```diff
@@ -11,9 +11,10 @@ monte_carlo:
       # TODO: Support inputs to simulate outages of biggest power plant "generators.p_nom.max()": [-1000MW 0MW]
       loads_t.p_set: [0.9, 1.1]
       generators_t.p_max_pu.loc[:, network.generators.carrier == "wind"]: [0.5, 1.2]
-      generators_t.p_max_pu.loc[:, network.generators.carrier == "solar"]: [0.5, 2]
+      # generators_t.p_max_pu.loc[:, network.generators.carrier == "solar"]: [0.5, 2]
```

Comment:
@GbotemiB Not sure both what is the purpose of both `generators_t.p_max_pu` lines, and what is a specific reason to comment-out solar generators? Would is possible to comment-out also the wind line, eventually adding some clarification for the both lines? 

I recognise that it's not very likely to have any effect on functionality, but that is always better to keep the code consistent.

Reply:
@ekatef, There is no specific reason for the comment-out on solar generators. It happened when I was iterating through the simulation. I think it will be if I remove the comment.